server/src/main/scala/org/dbpedia/extraction/server/stats/MappingStats.scala

```diff
@@ -24,23 +24,28 @@ class MappingStats(templateStats: TemplateStats, val templateName: String, val i
    * @param all count all properties or only the ones that are mapped
    * @param use count property use in wikipedia articles or just the definition in template
```

Comment:
Can you add the new parameter documentation here? and probably make invalid false by default?